tutorials/btp-sdm-gwi-s4hanacloud-create-com/btp-sdm-gwi-s4hanacloud-create-com.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Configure Outbound SAML OAuth between SAP S/4HANA Cloud, public edition and SAP BTP
+description: In this tutorial, you'll learn how to establish an outbound SAML OAuth configuration from SAP S/4HANA Cloud, public edition to SAP BTP.
+auto_validation: true
+time: 15
+tags: [ tutorial>beginner, software-product>sap-business-technology-platform, software-product>sap-s-4hana, software-product>sap-s-4hana-cloud]
+primary_tag: software-product>sap-business-technology-platform
+author_name: Vikram Kulkarni
+author_profile: https://github.com/Vikramkulkarni01
+---
+
+## Prerequisites
+ - You've access to SAP S/4HANA Cloud, public edition with admin privileges.
+ - You've created a service instance and service key of Document Management Service, Integration Option. For more information, see [Create a Service Instance and then a Service Key of SAP Document Management Service, Integration Option](btp-sdm-gwi-create-serviceinstance).
+ - From the generated service key, you copied the **`ecmservice`** URL. Your `ecmservice` URL would look something like `https://api-xyz-dm-gw.cfapps.sap.hana.ondemand.com/`.
+ - You've copied the `clientid`, `clientsecret`, and `tokenurl` parameters generated in the same subaccount of Cloud Foundry environment where Document Management Service, Integration Option instance is created.
+
+
+### You will learn
+ - How to use the existing communication scenario `SAP_COM_0190`
+ - How to add details from the service key generated from SAP BTP to SAP S/4HANA cloud, public edition
+ - How to add the details that are captured from SAP BTP in SAP S/4HANA Cloud, public edition
+---
+
+[ACCORDION-BEGIN [Step 1: ](Create Communication Arrangements)]
```

Comment:
"communication arrangement" is lower case (except when referring to UI) -- sentence case